admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -925,18 +925,30 @@ var form = (function() {
         e.preventDefault();
         send();
       });
+      jwerty.key('cmd+S', function(e) {
```

Comment:
The idea is good, but we have duplicated code ... Can you please merge the Mac hotkeys in the existing call ?

```js
jwerty.key('ctrl+S/cmd+S', function(e) {
```